conformance/Makefile

```diff
@@ -1,4 +1,5 @@
 TAG = edge
+VERSION = 1.1.0 # NGF version
```

Comment:
This variable is used in the conformance tests. It must've been removed by accident in a prior PR.